src/MLOps.NET.Azure/Storage/CosmosEntityConfigurator.cs

```diff
@@ -24,10 +24,12 @@ public static void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<RegisteredModel>().ToContainer(nameof(RegisteredModel));
             modelBuilder.Entity<DeploymentTarget>().ToContainer(nameof(DeploymentTarget));
             modelBuilder.Entity<Deployment>().ToContainer(nameof(Deployment));
+            modelBuilder.Entity<DataDistribution>().ToContainer(nameof(DataDistribution));
 
             modelBuilder.Entity<Data>().OwnsOne(x => x.DataSchema, dataSchema =>
             {
                 dataSchema.OwnsMany(x => x.DataColumns);
+                dataSchema.OwnsMany(x => x.DataColumns).OwnsMany(x => x.Distribution);
```

Comment:
I think an entity can't both own an entity e.g. DataColumns  -> owning Distribution and it also being it's own container (we will have to pick one). Given that it's logged separately from the Data, it may make sense that it is only in its own container

Reply:
I would be open to have it owned by a DataColumn too to be honest. I'll leave that for you to decide

Reply:
I am happy with either, depends on how it will be used. If we have it separate then we would need a GetDistrubution mehod as well right?

Reply:
Exactly. I think keeping it as it's own entity makes sense for now. We should probably also make sure that any `GetData` method also loads these instances